twilight-model/src/guild/afk_timeout.rs

```diff
@@ -0,0 +1,108 @@
+use crate::visitor::U16EnumVisitor;
+use serde::{
+    de::{Deserialize, Deserializer},
+    ser::{Serialize, Serializer},
+};
+use std::time::Duration;
+
+#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub enum AfkTimeout {
```

Comment:
shouldn't we design this more like other integer enums? https://github.com/twilight-rs/twilight/blob/0e92647560d1638640133e82f5bf67ab05eeaf3c/twilight-model/src/user/connection_visibility.rs#L3-L10

Reply:
Would probably be better 🤔 I've just taken the implementation from thread auto archive duration.

Reply:
When using a repr enum, we cannot have an `Unknown(u16)` variant.

Reply:
i don't think we use repr enums anymore for that purpose

Reply:
How else would you assign a number value to the variant 🤔 
`OneMinute` is `60` for example.

Reply:
We intentionally removed repr functionality in #1550 because it didn't support unknowns.